src/useRaf/useRaf.ts

```diff
@@ -0,0 +1,46 @@
+import { useState } from 'react';
+import { useIsomorphicLayoutEffect } from '..';
+
+/**
+ *
+ * @param ms milliseconds for how long to keep rendering component
+ * @param delay delay in milliseconds after which to start re-rendering component
+ * @returns number
+ */
+export function useRaf(ms = 1e12, delay = 0): number {
```

Comment:
`ms` should have no default value, also, it is better to rename in to `duration`